File: laramie-tesoreria-webapi/src/domain/entities/recaudadora.ts
export default class Recaudadora {

    id: number;
	codigo: string;
	nombre: string;
	orden: number;
	codigoCliente: string;
	idLugarPago: number;
	idMetodoImportacion: number;

	constructor(
        id: number = 0,
		codigo: string = "",
		nombre: string = "",
		orden: number = 0,
		codigoCliente: string = "",
		idLugarPago: number = 0,
		idMetodoImportacion: number = 0
	)
	{
        this.id = id;
		this.codigo = codigo;
		this.nombre = nombre;
		this.orden = orden;
		this.codigoCliente = codigoCliente;
		this.idLugarPago = idLugarPago;
		this.idMetodoImportacion = idMetodoImportacion;
	}

	setFromObject = (row) =>
	{
        this.id = row.id ?? 0;
		this.codigo = row.codigo ?? "";
		this.nombre = row.nombre ?? "";
		this.orden = row.orden ?? 0;
		this.codigoCliente = row.codigoCliente ?? "";
		this.idLugarPago = row.idLugarPago ?? 0;
		this.idMetodoImportacion = row.idMetodoImportacion ?? 0;
	}

}

File: modules/tesoreria/models/recaudadora.py
from sqlalchemy import Column, BigInteger, String, Integer

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class Recaudadora(Base):
    __tablename__ = "tesoreria_recaudadoras"

    id = Column(BigInteger, primary_key=True)
    codigo = Column(String(50), nullable=False)
    nombre = Column(String(250), nullable=False)
    orden = Column(Integer, nullable=False, default=0)
    codigo_cliente = Column(String(50), nullable=True)
    id_lugar_pago = Column(BigInteger, nullable=True)
    id_metodo_importacion = Column(BigInteger, nullable=True)
